src/macros.rs

```diff
@@ -68,6 +68,16 @@ macro_rules! redis_event_handler {
             $crate::raw::Status::Ok as c_int
         }
 
+        let all_available_flags = $crate::raw::get_keyspace_notification_flags_all();
+        if !all_available_flags.contains($event_type) {
+            let not_supported = $event_type.difference(all_available_flags);
+            log::warn!("The event notification flags set aren't supported: {not_supported:?}");
+
+            // $crate::Context::new($ctx).log_warning(&format!(
+            //     "The event notification flags set aren't supported: {not_supported:?}"
+            // ));
+        }
```

Comment:
We can use an else here no? Only register to the event if it is supported. This way we do not count of Redis to ignore it and not return an error. IIRC the `RedisModule_SubscribeToKeyspaceEvents` does return an success/failed indication so Redis might one day decide to validate this input.